domains/seven.json

```diff
@@ -6,7 +6,13 @@
         },
     
         "record": {
-            "CNAME": "7op3.github.io"
+            "CNAME": "7op3.github.io",
+        "MX": [
+                "mx1.simplelogin.co",
+                "mx2.simplelogin.co"
+        ],
+        
+        "TXT": ["sl-verification=ivfyrhmrpfrkrgrkywcghobxqkhynw","v=spf1 include:simplelogin.co ~all"]
```

Comment:
You can't use CNAME with other records (use the A records instead for github pages)

Reply:
I've changed it. Will it work now?